org.mixedrealitytoolkit.uxcore/Tests/Runtime/SliderTests.cs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using MixedReality.Toolkit.Input.Tests;
+using NUnit.Framework;
+
+namespace MixedReality.Toolkit.UX.Runtime.Tests
+{
+    using Core.Tests;
+    using Input;
+    using System;
+    using System.Collections;
+    using UnityEditor;
+    using UnityEngine;
+    using UnityEngine.TestTools;
+    using Object = UnityEngine.Object;
+
+    public class SliderTests : BaseRuntimeInputTests
+    {
+        private const string TouchSliderPrefabPath =
+            "Packages/org.mixedrealitytoolkit.uxcore/Tests/Runtime/Prefabs/TouchSliderTest.prefab";
```

Comment:
The UXCore package is not meant to have prefabs in it.  And I would hate to have to maintain two sets of prefabs (one in UXCore and one in UXComponents)

So please move this unit tests to UXComponents, and add use the official prefabs.